saleor/csv/utils/export.py

```diff
@@ -178,6 +239,136 @@ def export_products_in_batches(
         append_to_file(export_data, headers, temporary_file, file_type, delimiter)
 
 
+def export_tally_data_in_batches(
+    queryset: "QuerySet",
+    temporary_file: Any,
+    headers: List[str],
+    delimiter: str,
+    file_type: str
+):
+    for batch_pks in queryset_in_batches(queryset):
+        invoice_batch = Invoice.objects.filter(pk__in=batch_pks).prefetch_related(
+            "order",
+            "order__shipping_address"
+        )
+        export_data = list(invoice_batch.values(
+            reciever_nip=F("order__shipping_address__vat_id"),
+            receiver_name=Concat(F("order__shipping_address__first_name"),
+                                 Value(" "),
+                                 F("order__shipping_address__last_name")),
+            receiver_address=Concat(F("order__shipping_address__postal_code"),
+                                    Value(" "),
+                                    F("order__shipping_address__city"),
+                                    Value(", "),
+                                    F("order__shipping_address__street_address_1"),
+                                    Value(" "),
+                                    F("order__shipping_address__street_address_2")),
+            invoice_number=F("number"),
+            created_date=Cast(TruncDay('created', DateTimeField()), CharField()),
+            sale_date=Cast(TruncDay('created', DateTimeField()), CharField()),
+            WDT=Value("0"),
+            Export=Value("0"),
+            net_price_0_percent=Value("0"),
+            transport_price_5_percent=Value("0"),
+            tax_price_5_percent=Value("0"),
+            net_price_7_8_percent=Value("0"),
+            tax_price_7_8_percent=Value("0"),
+            net_price_22_23_percent=F("private_metadata__summary__to"),
+            tax_price_22_23_percent=F("private_metadata__summary__to")
+        ))
+        append_to_file(export_data, headers, temporary_file, file_type, delimiter)
+
+
+def get_query_values_list(
+        invoice_queryset: "QuerySet",
+        payment_queryset: "QuerySet",
+        is_correction: bool = False
+):
+    if is_correction:
```

Comment:
CorrectinType, File, BruttoParent might be null for regular invoice and we can get rid of "if is_correction" section.